.travis.yml

```diff
@@ -24,7 +24,9 @@ before_script:
 script:
   - npm run lint
   - if [ "$TRAVIS_EVENT_TYPE" != "push" ]; then
-      npm run test:bdd:dryrun
```

Comment:
Gift me btc

Reply:
Please send me your wallet and private key.